doc/user_guide/color_usage.rst

```diff
@@ -0,0 +1,1525 @@
+         :remove-code:
+
+         plot_scheme("category20b", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         category20c
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("category20c", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         dark2
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("dark2", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         paired
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("paired", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         pastel1
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("pastel1", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         pastel2
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("pastel2", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         set1
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("set1", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         set2
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("set2", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         set3
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("set3", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         tableau10
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("tableau10", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+   * - .. code-block:: none
+    
+         tableau20
+
+     - .. altair-plot::
+         :remove-code:
+
+         plot_scheme("tableau20", catg_schemes, cvd=True, continuous=False).configure_scale(rectBandPaddingInner=0.1)
+
+Palettes from other sources
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This subsection focuses on pre-designed color palettes that can be
+used in data visualization from other sources.
+
+Here some color schemes from Paul Tol, https://personal.sron.nl/~pault/ which are picked as being:
+
+- distinct for all people, including colour-blind readers
+- distinct from black and white
+- distinct on screen and paper
+- matching well together
+
+.. altair-plot::   
+    :output: none
+
+    tol_schemes = {
+        "tol_bright" : ['#4477AA', '#EE6677', '#228833', '#CCBB44', '#66CCEE', '#AA3377', '#BBBBBB'],
```

Comment:
This might need some more explanation on how to use these custom palettes. Maybe the section could start with an intro of how to make your own palette (like what is in the color customization section right now) and then show some from this other source. Also, I am not familiar with Paul Tol, but maybe we can think about the presentation of his palettes so that it doesn't seem like the properties listed above are not true for the built-in altair color schemes, because in many cases they are.